roles/testnode/tasks/apt_systems.yml

```diff
@@ -1,6 +1,14 @@
 ---
 - name: Setup local repo files.
   include: apt/repos.yml
+  when: ansible_distribution_major_version < 15
```

Comment:
Good catch, I'll need to limit this to ubuntu as well.  We only support debian 7 right now, so it would n't be an issue, but better to be explicit anyway though.